configure.ac

```diff
@@ -86,7 +86,7 @@ static char buf[1024];]],
 AC_LANG_POP(C++)
 
 
-AC_CHECK_FUNCS([statvfs pipe2])
+AC_CHECK_FUNCS([statvfs pipe2 close_range])
```

Comment:
Still getting this:

src/libutil/unix/file-descriptor.cc:127:20: error: use of undeclared identifier 'SYS_close_range'
  127 |     return syscall(SYS_close_range, first, last, (unsigned int)flags);

Reply:
Nevermind. My autoconf was stale.